cabal-install/changelog

```diff
@@ -1,6 +1,8 @@
 -*-change-log-*-
 
 3.0.0.0 (current development version)
+	* `v2-repl` no longer changes directory to a randomized temporary folder
```

Comment:
Note: the 3.0 branch is cut, but this PR is against `master`.

@23Skidoo should we introduce "needs-backport-to-3.0" label to remember cherry-pick those?

Reply:
I missed that! Should this be backported, or should I just move the changelog entry?

I feel like the QoL gain is pretty big. 🙂

Reply:
@phadej We already have `attention: needs-backport`.

Reply:
@typedrat I can defo cherry-pick this one.